src/lwc/rd2EntryFormDonorSection/rd2EntryFormDonorSection.html

```diff
@@ -25,30 +25,33 @@
 
                     <template if:true={isContactDonor}>
                         <lightning-layout-item padding="horizontal-small" size="12" small-device-size="6" if:true={fields.contact} >
-                            <lightning-input-field field-name={fields.contact.apiName} value={contactId} required tabindex="0">
+                            <lightning-input-field 
+                                field-name={fields.contact.apiName} 
+                                value={contactId} 
+                                required tabindex="0" 
+                                onchange={handleContactChange}>
```

Comment:
Very minor, but how about `handleDonorChange` since the same event is used for both the Account and Contact?

Reply:
It is only used for the Contact lookup field value change, not Account lookup field value change. If there is confusion when reading the code, let me know where and I will fix it.

Reply:
Oh, I saw the line below that said "isAccountDonor" and thought that the input was for the Account, but it's still for the Contact, so you can ignore this.

Reply:
I changed code a bit so Established Date field is constructed only twice. It is still difficult to see there are two Contact fields, could not make any change to make it as one.